### PR TITLE
flow fixes

### PR DESCRIPTION
**Pull request by TomPridham**

### CODE REVIEW

File: src/write-json.js [hunk 1]
```diff
@@ -14,21 +14,23 @@ const writeJson = (
   indent?: number | (?Error) => void = 2,
   cb?: (?Error) => void
 ): void => {
+  let callback = cb
+  let ind
```
Comment (TomPridham) [inline src/write-json.js]: im not sure if this will work or not

Comment (TomPridham) [reply src/write-json.js]: nvm., fixed it

File: src/lighten-or-darken.js [hunk 2]
```diff
@@ -17,7 +17,6 @@ const lightenOrDarken = (col: string, amt: number): string => {
   let g: number = (num & 0x0000FF) + amt
 
   if (col[0] === '#') {
-    col = col.slice(1)
```
Comment (TomPridham) [inline src/lighten-or-darken.js]: this line wasn't used unless it was causing some side effect somewhere that the function was relying on

Comment (zacanger) [reply src/lighten-or-darken.js]: This line does actually matter, but it's in the wrong place. I'll fix it after merging.

**Pull request merged**